src/App.tsx

```diff
@@ -1,14 +1,56 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { Select } from './types/Select';
 
 export const App: React.FC = () => {
+  const [todoList, setTodoList] = useState<Todo[] | null>(null);
+  const [selectedTodo, setTodoModal] = useState<Todo>();
+  const [user, setUser] = useState<User | null>(null);
+  const [inicializationModal, setInicModal] = useState(false);
```

Comment:
1. follow the convention for the boolean variable names
2. if state is inicializationModal setState should be setInicializationModal